src/contexts/modal/ModalContext.tsx

```diff
@@ -194,11 +194,14 @@ function ModalProvider({ children }: Props) {
 
     const globalNavbar = document.querySelector('.GlobalNavbar') as HTMLElement | null;
     if (globalNavbar) {
-      globalNavbar.style.transform = modalShowing ? 'translateX(0px)' : 'unset';
-
-      globalNavbar.style.width = modalShowing
-        ? `calc(100vw - ${currentScrollbarWidth}px)`
-        : 'unset';
+      /**
+       * Commenting this out to prevent a regression on the banner width not extending to 100%.
+       * Leaving the code in tho in case scrollbar issue comes back
+       */
+      // globalNavbar.style.transform = modalShowing ? 'translateX(0px)' : 'unset';
+      // globalNavbar.style.width = modalShowing
+      //   ? `calc(100vw - ${currentScrollbarWidth}px)`
+      //   : 'unset';
```

Comment:
this code was causing this regression:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/12162433/191840110-caebc2c2-3f8f-4ac7-a15e-7879982ccb9e.png">

commenting it out doesn't seem to impact scrollbar issues when a modal is open